CIP-xxxx/README.md

```diff
@@ -0,0 +1,127 @@
+---
+CIP: ?
+Title: SRV registry
+Status: Proposed
+Category: Network
+Authors:
+  - Marcin Szamotulski <marcin.szamotulski@iohk.io>
+Implementors: N/A
+Discussions: 
+  - [PR](https://github.com/cardano-foundation/CIPs/pull/1033)
+Created: 2025-04-22
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP defines the procedure for assigning Service Record (SRV) prefixes for Cardano applications (like `cardano-node`, `mithril`, etc.).
+
+This CIP constructs means of sharing authoritative information between SPOs who deploy services and DAP developers who write them.
+
+## Motivation: why is this CIP necessary?
+
+The **Cardano Ledger** allows the use of SRV records in the **SPO** registration certificate (see [register-stake-pool]).
```

Comment:
```suggestion
The **Cardano Ledger** allows the use of SRV in the **SPO** registration certificate (see [register-stake-pool]).
```
Since SRV includes abbreviation for 'record' do we need to say it again?
perhaps this is the convention, that I am unfamiliar with

Reply:
It's often used this way.  Some examples: [_What is a DNS SRV record?_](https://www.cloudflare.com/learning/dns/dns-records/dns-srv-record/), or the original [RFC](https://www.rfc-editor.org/rfc/rfc2782)